### ISSUE TITLE
http://www.cmb.usc.edu/people/andrewds/rmap link broken

### ISSUE THREAD

**Issue by peterjc** [smithlabcode/rmap#14]
The project URL http://www.cmb.usc.edu/people/andrewds/rmap listed in http://www.ncbi.nlm.nih.gov/pubmed/19736251 is broken.

Could you get it to redirect here (GitHub) or to http://rulai.cshl.edu/rmap/ instead?

**Comment by peterjc** [comment]
Also the source code link on http://rulai.cshl.edu/rmap/ is still http://www.cmb.usc.edu/people/andrewds/rmap/rmap_v2.05.tbz2 which redirects (301 Moved Permanently) to http://dornsife.usc.edu/cbb/people/andrewds/rmap/rmap_v2.05.tbz2 which fails (Error 404 Not Found).

If there was a tag for v2.05 on GitHub that might work as an alternative?

**Comment by andrewdavidsmith** [comment]
Thanks Peter, I'll see what I can do.

**Comment by saketkc** [comment]
Peter also pointed out a cached version which is here:
https://depot.galaxyproject.org/package/linux/x86_64/rmap/

**Comment by peterjc** [comment]
@saketkc Yes, but unfortunately the Galaxy Project cache does not seem to have the source code tar ball, which is how I spotted the 404 link.

**Comment by saketkc** [comment]
Indeed, I missed out on the fact that they are hosted binaries.

**Comment by peterjc** [comment]
@saketkc found http://smithlabresearch.org/downloads/rmap_v2.05.tbz2 for the source code I was looking for.

Perhaps the original published URL http://www.cmb.usc.edu/people/andrewds/rmap could be redirected to http://smithlabresearch.org/software/rmap/ ?
